### ISSUE TITLE
Instruction to disable telemetry is broken

### ISSUE THREAD

**Issue by chris-morgan** [yarnpkg/yarn#8882]
```
➤ YN0065: Yarn will periodically gather anonymous telemetry: https://yarnpkg.com/advanced/telemetry
➤ YN0065: Run yarn config set --home enableTelemetry 0 to disable
```

This is also the command shown at https://yarnpkg.com/advanced/telemetry/ (“To disable it on your whole machine (but not for anyone else), run `yarn config set --home enableTelemetry 0`.”).

So I ran this:

```console
$ yarn config set --home enableTelemetry 0
yarn config v1.22.19
success Set "0" to true.
Done in 0.08s.
```

I followed a hunch from https://classic.yarnpkg.com/en/docs/cli/config which shows `--global` at the *end*, and that seemed to do better:

```console
$ yarn config set enableTelemetry 0 --home
yarn config v1.22.19
success Set "enableTelemetry" to "0".
Done in 0.07s.
```

**Comment by ondratra** [comment]
The problem is the running of old yarn version `1.x.x`. 
**TL;DR: Run the command from a repository where Yarn v3 (berry) is installed.**

--- 

The thing is that on most (?) systems, the `yarn` that is installed globally with the package manager is of version `1.x.x`, and version `3.x.x` is installed to various project repositories via `yarn set version berry` or similar.
Thanks to that, running `yarn` in a repository with the latest Yarn version will yield different results than running it outside of any repository.

So if we assume the Debian system, the following is true:
```
$ cd / # move somewhere out of any repository
$ yarn -v
1.22.5
$ yarn config set --home enableTelemetry false # this doesn't yield the desired result
success Set "false" to true.
Done in 0.02s. 

$ mkdir ~/myTestRepo
$ cd ~/myTestRepo
$ yarn init -y
$ yarn set version berry
$ yarn -v
3.4.1
$ yarn config set --home enableTelemetry false
➤ YN0000: Successfully set enableTelemetry to false
```

I can add that when you run `yarn` v1:
- `--home` argument is effectively ignored - running `yarn config set enableTelemetry false` has the same effect as `yarn config set enableTelemetry false --home` that can be checked via `yarn config list`.
- `--home` argument is effectively ignored in the same way when using `yarn config list --home`
- if you run the bad command `yarn config set --home enableTelemetry 0` you can reverse the effects on your system with `yarn config delete "0" && yarn config delete "false"`

--- 
This particular GitHub issue should, IMHO, be solved by updating [Yarn documentation](https://yarnpkg.com/advanced/telemetry) with a hint about this reality. 

It is more easily achievable than marking Yarn v1 obsolete and replacing its packages in major OS distributions.
